src/ebay-tabs/tab-panel.tsx

```diff
@@ -24,7 +26,7 @@ const TabPanel: FC<TabPanelProps> = ({
             {...rest}
             aria-labelledby={`default-tab-${index}`}
             className={classNames(className, 'tabs__panel')}
-            id={`default-tabpanel-${index}`}
+            id={`${parentId || 'default'}-tabpanel-${index}`}
```

Comment:
would prop id also work (to avoid more unique props )
            id={`${prop.id || 'default'}-tabpanel-${index}`}

Reply:
I thought about using `prop.id` just like what you proposed, but that would diverge on how `id` prop normally works since the tab panel will output different id than the given one. I think we should keep `id` behaviour the same on every places, which will render the given id on the output HTML (see our checkbox, tooltip content, listbox button, ...). Component id might be used by `aria-labelledBy`, `aria-controls`, label's `for`, or just for `<a>` anchor.

But I see your point of not introducing a new prop `parentId` just for this purpose. We can use tab panel's id prop but we need to move the id generation logic inside tab panel into the parent component tabs. With this way, the tabs will decide what id its children will be and the tab panel will render the given id as-is.